<ui version="4.0" >
 <class>IndexingDialog</class>
 <widget class="QDialog" name="IndexingDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>169</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Indexing Parameters</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>Accuracy</string>
   </property>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>L</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Deg.</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>100</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>mm</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>120</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>H</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Search Range</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Angular Tol.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>Spot Accuracy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>K</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxH" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>30</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxK" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>30</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxL" >
    <property name="geometry" >
     <rect>
      <x>270</x>
      <y>30</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="angTol" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>70</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="spotAccuracy" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>100</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
    <property name="alignment" >
     <set>Qt::AlignRight</set>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>30</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IndexingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IndexingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
